strutil/strutil.go

```diff
@@ -70,3 +70,33 @@ func Quoted(names []string) string {
 
 	return strings.Join(quoted, ", ")
 }
+
+// WordWrap takes a input string and word wraps after `n` chars
+// into a new slice.
+//
+// Caveats:
+// - If a single word that is biger than max will not get wrapped
+// - Extra whitespace will be removed
+func WordWrap(s string, max int) []string {
+	n := 0
+
+	out := []string{}
```

Comment:
Usual way is with []string(nil) or var syntax so we're not allocating the empty backing array. Trivial in this context, though.